composer.json

```diff
@@ -39,15 +39,21 @@
 
         "sonata-project/cache-bundle": "^2.1",
         "sonata-project/doctrine-phpcr-admin-bundle": "^2.0",
-        "sonata-project/admin-bundle": "^3.2",
+        "sonata-project/admin-bundle": "3.2 as ^3.6.0",
```

Comment:
@ElectricMaxxx why this as? Why not doing `"sonata-project/admin-bundle": "^3.6"`? This still gives us SonataAdminBundle 3.2, which doesn't work witht the SonataIntegrationBundle.